clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ResolvedChain.java

```diff
@@ -73,7 +71,7 @@ public Chain getCompactedChain() {
 
     @Override
     public boolean isCompacted() {
-      return compacted;
+      return compactedChain != null;
```

Comment:
Please no, let's not have compactedChain being `null` legal. What is it buying us exactly?